src/styles/core-buttons.scss

```diff
@@ -0,0 +1,28 @@
+.wp-block-button {
+
+	&.is-share-button {
+		.wp-element-button {
+			cursor: pointer !important;
+			transition: background-color 200ms ease-in;
+			position: relative;
+			pinter-events: all;
+		}
+
+		&.copied .wp-element-button {
+
+			&:after {
+				position: absolute;
+				left: 0;
+				right: 0;
+				top: 0;
+				bottom: 0;
+				background-color: var( --wp--preset--color--wmf-report-green, #318557 ) !important;
+				color: #fff;
+				content: "Copied!";
```

Comment:
Not necessary for this project but a trick I like to use is to use a filter to write a data-attr onto the element with localized text, which we can then use in the `content:` rule with the `attr()` css function. Not needed this time though!

Reply:
oh, that's good. I'll use that in the future.